format/avro/decoders/null.go

```diff
@@ -0,0 +1,16 @@
+package decoders
+
+import (
+	"github.com/wader/fq/format/avro/schema"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func decodeNullFn(schema schema.SimplifiedSchema, sms ...scalar.Mapper) (DecodeFn, error) {
+	// null is written as zero bytes.
+	return func(name string, d *decode.D) interface{} {
+		// Is this the best way to represent null in fq?
```

Comment:
Good question, i should give this some thought, have the same issue in other decoders

Reply:
Now there more proper nil support #81, it was kind of support in interp but i forgot about for the decoder as non of the earlier decoded needed it.

Reply:
Thanks, updated to use FieldValueNil